### PR TITLE
Add role for metrics gathering

### PR DESCRIPTION
**Pull request by RDiMag**
Roles directory for tasks associated with getting AWS metric-statistics using the Aurora instance ID.

### CODE REVIEW

**Comment by mw2q** [comment]
Ok.  Let do:
*  get-aws-cloudwatch-aurora-postgres-stats because there is aurora mysql etc
* make main.yml empty because i can't think of anything the role should do just by importing it
* and the other renames as agreed

File: roles/get-aws-cloudwatch-aurora-postgres-stats/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - ReplicationSlotDiskUsage
+    - StorageNetworkReceiveThroughput
+    - StorageNetworkThroughput
+    - StorageNetworkTransmitThroughput
+    - SwapUsage
+    - TransactionLogsDiskUsage
+    - WriteIOPS
+    - WriteLatency
+    - WriteThroughput
+
```
Comment (mw2q) [inline roles/get-aws-cloudwatch-aurora-postgres-stats/vars/main.yml:33]: I didn't notice this previously.  See if you can find a setting in your editor to always make sure there is always an EOF at the end of the file.

**Review by mw2q — CHANGES_REQUESTED** [review]
Looks good.  Just that one EOF that should be a simple fix.

File: roles/get-aws-cloudwatch-aurora-postgres-stats/vars/main.yml (hunk 1 above)
Comment (RDiMag) [reply roles/get-aws-cloudwatch-aurora-postgres-stats/vars/main.yml]: I believe I resolved this.

File: roles/get-aws-cloudwatch-aurora-postgres-stats/tasks/get-end-time.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        - "ansible_date_time"
+
+  - name: Get timestamp from the system
+    shell: "date +%Y-%m-%d-%H:%M:%S"
+    register: tstamp
+
+  - name: Get end time
+    set_fact:
+      end: "{{ tstamp.stdout }}"
+
```
Comment (mw2q) [inline roles/get-aws-cloudwatch-aurora-postgres-stats/tasks/get-end-time.yml:14]: I see one more missing EOF here. :)

**Review by mw2q — CHANGES_REQUESTED** [review]
I think that looks good.  I'm sure you can fix the EOF thing quickly.  Then go ahead and squash this back into a single commit, then I think we can merge it.

**Pull request merged**